streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkTrackingTask.java

```diff
@@ -271,7 +271,8 @@ private void getFromFlinkRestApi(Application application, StopFrom stopFrom) thr
         FlinkCluster flinkCluster = getFlinkCluster(application);
         JobsOverview jobsOverview = httpJobsOverview(application, flinkCluster);
         Optional<JobsOverview.Job> optional;
-        if (ExecutionMode.isYarnMode(application.getExecutionMode())) {
+        if (ExecutionMode.isYarnMode(application.getExecutionMode()) &&
```

Comment:
change it to this, will be more clear : `
if (ExecutionMode.YARN_APPLICATION.equals(application.getExecutionMode()) || ExecutionMode.YARN_PER_JOB.equals(application.getExecutionMode())) 
`